src/components/PersonalInfoForm.css

```diff
@@ -0,0 +1,50 @@
+.personal-form-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-bottom: 20px;
```

Comment:
nit: I think usually it's better to use rem rather than px for margins

Reply:
Replaced those for rem